### DISCUSSION TITLE
In Vercel, How can I create a consumer that listens to messages in a message queue (AWS SQS)?

### DISCUSSION THREAD

**Discussion by hyunwoona** [vercel/next.js#44108] [Help]
Hi, I am using a service called getstream.io, to generate social media feeds.
They provide a way to send notifications to an AWS SQS queue. All I need to do is provide the AWS SQS URL.

However, I need to listen to the messages in SQS, (potentially aggregate them over a day or so, e.g. your notifications for December 17th) and send a notification email to users.
How can I do this in Vercel? Serverless functions only live during the lifetime of the request and response, but this will need to keep running.
I really want to avoid spinning up another server just for this.

**Comment by deadcoder0904** [comment]
@hyunwoona did you find a solution?
